examples/neo-place/README.md

```diff
@@ -0,0 +1,25 @@
+# Neo/Place
+
+## Getting Started
+
+1. Setup a .env file with the neo4j credentials:
```

Comment:
I think it would be good to mention briefly that there are two options, AMQP and local single instance. And maybe a few words on how to set those two up. I know it's partly mentioned already but I think we can make it a bit easier and clearer for the people who want to try this super cool application themselves. Wdyt?